tests/e2e/e2e_suite_test.go

```diff
@@ -226,8 +228,9 @@ func (s *Suite) TearDownSuite() {
 func (s *Suite) TestDCGMExporter() {
 	s.DCGMExporterPrechecks()
 
-	s.T().Run("Create workload pod", func(t *testing.T) {
-		t.Log("Creating a workload pod...")
```

Comment:
Refactoring, to use the testify Suite framework to run sub-tests, which makes much easier work with the tree of tests.